.circleci/config.yml

```diff
@@ -41,7 +41,9 @@ jobs:
 
       - run:
           name: Build
-          command: docker-compose build web
+          command: |
+            docker-compose build
+            docker-compose pull db
```

Comment:
What does that do? I guess it happens automatically by `docker-compose up -d web` but I guess by doing it here you're "preparing" to have the download done so it doesn't take so damn long to start the web container thing later. 

Is that right? In fact, if it is right I think you should jot this down as a comment. At least I had a reaction of "Why is this done???"

Reply:
You're right, this is to get all the prep work out of the way first thing, so that the test steps are "purer", and have more consistent timing. I'll add a comment.